src/main.js

```diff
@@ -5,7 +5,10 @@ import App from './App'
 import router from './router'
 import store from './store'
 
+window.tn = t.bind(this, 'notes')
```

Comment:
why is that?
It's usually a good behaviour to not pollute the window with unknown variables (for other apps/scripts) :)

Reply:
You're right. It was just meant as a continuation of

https://github.com/nextcloud/server/blob/f429098a728783e08244a60025a8c0cda25bf210/core/src/globals.js#L111

but it is not really required to be global. I will remove this resp. move this to `Vue.prototype.tn = t.bind(this, 'notes')`. Would that be okay?

Reply:
Sure! :)
But I'm still curious about what is the tn function :wink:

Reply:
**t**ranslate (for) **n**otes? :)

An alternative to mangling the Vue prototype is a mixin: https://blog.wuc.me/2018/10/22/mixing-nextcloud-globals-into-vue.html

Reply:
Yes, correct. It's just a shortcut for `t('notes', ...)`. Now, I can write `tn(...)`. 😁 

@ChristophWurst Ah, yes, a global mixin would be more the correct Vue-way. I think I will switch to this.